src/sentry/static/sentry/app/views/admin/adminQueue.tsx

```diff
@@ -5,7 +5,20 @@ import {Panel, PanelHeader, PanelBody} from 'app/components/panels';
 import InternalStatChart from 'app/components/internalStatChart';
 import {SelectField} from 'app/components/forms';
 
-export default class AdminQueue extends AsyncView {
+const TIME_WINDOWS = ['1h', '1d', '1w'] as const;
+
+type TimeWindow = typeof TIME_WINDOWS[number];
+
+type State = AsyncView['state'] & {
+  timeWindow: TimeWindow;
+  since: number;
+  resolution: string;
+  taskName: string;
+  activeTask: string;
+  taskList: string[];
```

Comment:
```suggestion
  timeWindow: TimeWindow | null;
  since: number | null;
  resolution: string | null;
  taskName: string | null;
  activeTask: string | null;
  taskList: string[] | null;
```